packages/creamy_field/example/lib/main.dart

```diff
@@ -51,20 +51,11 @@ class _MyEditorAppState extends State<MyEditorApp> {
           FlatButton(
             child: Text('Add tab'),
             onPressed: () {
-              // Adds a tab at the selection's base base-offet
               controller.addTab();
             },
           )
         ],
       ),
-      // A CreamyField is a text field which supports CreamyEditingController
-      // & CreamySyntaxHighlighter.
-      //
-      // This text field is similar to Material TextField but will be
-      // tailored (expected in future) for writing rich text,
-      // especially for markup & programming languages.
-      //
-      // For now, using it like Material TextField as below
```

Comment:
The first two lines look correct to me in this comment, rest is obsolete but you removed everything. Was something incorrect or misleading here?